src/components/Story.js

```diff
@@ -76,6 +76,7 @@ const Story = () => {
       minDetectionConfidence: 0.5,
       minTrackingConfidence: 0.5,
       selfieMode: true,
+      refineFaceLandmarks: true,
```

Comment:
interesting -- I didn't realize this was part of the API, will have to read into it!